xc7/primitives/bram/rambfifo36e1.pb_type.xml

```diff
@@ -178,13 +186,60 @@ The RAM has extra bits that can be used for parity (DIP / DOP).
    <input  name="CASCADEINB"    num_pins="1"  />
    <output name="CASCADEOUTA"   num_pins="1"  />
    <output name="CASCADEOUTB"   num_pins="1"  />
+   <metadata>
+     <meta name="fasm_features">
```

Comment:
Most of this should be a fasm_params block with the parameters being sourced from the eblif?

Reply:
I'm just trying to get something working for now, so I copied these features from Vivado using bit2fasm.

Reply:
Sure, but do it via directly setting RAMB36E1_VPR parameters, rather than manipulating the XML.  You have to do it eventually via the techmap, but let VPR just paste the RAMB36E1_VPR parameters, and change the Verilog.